libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Extensions/CollectionExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Integration.Runtime.Extensions
+{
+    /// <summary>
+    /// Defines extension methods for <see cref="ICollection{T}"/>.
+    /// </summary>
+    internal static class CollectionExtensions
```

Comment:
Making it private would make it untesteable. Is that really better? In exchange for that this can be deleted in favor of adding 3 lines to ServiceCollectionExtensions

Reply:
Initially it was used in more places I think, but I just deleted it and resulted in 1 line of code and 2 braces so I think its better to own less code.